server/src/actors/project.rs

```diff
@@ -334,8 +342,13 @@ impl ProjectState {
 
     /// Returns whether this state is outdated and needs to be refetched.
     pub fn outdated(&self, config: &Config) -> bool {
+        let jitter = config.cache_timeout_jitter();
+        let factor = jitter * (self.seed * 2.0 - self.seed);
```

Comment:
isn't this just `factor = jitter * self.seed` or are you trying to get rid of float glitches this way?